plume/src/main/kotlin/io/github/plume/oss/Extractor.kt

```diff
@@ -339,7 +339,6 @@ class Extractor(val driver: IDriver) {
 
     private fun getMethodsAndBuildBodies(cs: Set<SootClass>): Set<SootMethod> {
         return cs.flatMap { it.methods }
-            .mapNotNull { runCatching { it.retrieveActiveBody() }.getOrNull()?.method }
```

Comment:
This looks like a line removed from a merge/rebase issue. But this is important as some application classes like abstract ones may throw exceptions here.

Reply:
You need to make the call to `it.retrieveActiveBody` otherwise `hasActiveBody` will often return false even if there may be a body one can create

Reply:
Confirmed.

Reply:
`it.retrieveActiveBody()` will throw uncaught exceptions here. I suggest splitting this stream into:
```kotlin
return cs.flatMap { it.methods }
            .asSequence()
            .mapNotNull { runCatching { it.retrieveActiveBody() }.getOrNull()?.method }
            .filter { it.hasActiveBody() || it.isNative }
```

Reply:
Does Native Method has activeBodies?

Reply:
To make commit clear, I will remake PR.

Reply:
I use `https://github.com/plume-oss/benchmarking` to measure how Plume does against some large open-source projects and I see exceptions thrown on active bodies sometimes on classes from the given JAR. These seem to be in abstract classes.